src/game/character-init/initCharacter.js

```diff
@@ -0,0 +1,14 @@
+const Classes = require('./initClasses')
+
+const classList = Classes.initClasses
+
+module.exports = {
+    // entry point to create a class with associated logic. Called from createCharInstance
+    getCharWithClass : ( className, name, gender ) => {
+        for ( var key in classList ) { 
```

Comment:
Varssssssssss

Reply:
Zeker voor een variabele naam als 'key' is dat echt dangerous

Reply:
Zou hier sws een .filter van maken

Reply:
En zou ook altijd ipv key een descriptieve variabele naam gebruiken, bv class, geizen je over classes itereert